apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -84,19 +85,42 @@ class LessonOverview extends Component {
     hasVerifiedResources: PropTypes.bool.isRequired
   };
 
+  recordAndNavigateToPdf = (e, firehoseKey, url) => {
+    // Prevent navigation to url until callback
+    e.preventDefault();
```

Comment:
if you remove the href can you remove this?

Reply:
I want the href so that the link can be right clicked to open in a new tab. But yeah, I need the `preventDefault` because `href` is still set.